libraries/AP_NavEKF2/AP_NavEKF2.h

```diff
@@ -485,4 +493,9 @@ class NavEKF2 {
     // new_primary - index of the ekf instance that we are about to switch to as the primary
     // old_primary - index of the ekf instance that we are currently using as the primary
     void updateLaneSwitchPosDownResetData(uint8_t new_primary, uint8_t old_primary);
+
+    uint8_t core_initial_imu[8];
```

Comment:
Can we please make sure when adding new class variables that there is s description in the comments as per the rest of the code. As a minimum it should indicate what that block of variable is used for.